base-lite/pom.xml

```diff
@@ -0,0 +1,97 @@
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  ~-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>io.confluent</groupId>
+        <artifactId>common-docker</artifactId>
+        <version>7.4.0-0</version>
+    </parent>
+
+    <packaging>pom</packaging>
+
+    <artifactId>cp-base-lite</artifactId>
+
+    <description>Base for new Confluent lightweight Docker images</description>
+    <name>${project.artifactId}</name>
+
+    <properties>
+        <docker.skip-build>false</docker.skip-build>
+        <docker.skip-test>false</docker.skip-test>
+        <docker.pull-image>true</docker.pull-image>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>io.confluent</groupId>
+            <artifactId>utility-belt</artifactId>
+            <version>${io.confluent.common-docker.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <version>${junit.version}</version>
+            <scope>test</scope>
+        </dependency>
+    </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <artifactId>maven-resources-plugin</artifactId>
+                <version>${maven-resources-plugin.version}</version>
+                <executions>
+                    <execution>
+                        <id>copy-resources</id>
+                        <phase>compile</phase>
+                        <goals>
+                            <goal>copy-resources</goal>
+                        </goals>
+                        <configuration>
+                            <overwrite>true</overwrite>
+                            <outputDirectory>target</outputDirectory>
+                            <resources>
+                                <resource>
+                                    <directory>${basedir}/../docker-utils/target</directory>
+                                    <includes>
+                                        <include>docker-utils-${CONFLUENT_VERSION}-jar-with-dependencies.jar</include>
+                                    </includes>
+                                </resource>
+                            </resources>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>com.spotify</groupId>
+                <artifactId>dockerfile-maven-plugin</artifactId>
+                <configuration>
+                    <buildArgs>
```

Comment:
The analogy is same as above. We don't need anything on top of AK. However, we will get it double checked. 
@mkcops would you be able to help to point this to appropriate person. The context is we need a kafka lite image wihthout any add-ons. Thanks!

Reply:
@nsharma-git This isn't an AK image and we don't release an image with AK. I don't consider SSL to be a add-on feature. We have onboarding and tutorials that instruct people to use SSL. So these images won't work when new users try to do the basic tutorials if you remove open SSL.

Reply:
By AK, I mean AK equivalent bits in confluent. These debs/rpms are already packed with all required jars for ssl/kerberos use cases. 
In case you are referring to Openssl, Keys and certs would be generated by user on any user's machine. Out tutorials guides them to install Open SSL and generate keys and certs ( for self signed setup )